.eslintrc

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
This is the configuration file for [eslint](http://eslint.org/).
It is used by `eslint-loader` for `webpack` in order to lint the Javascript code.